pom.xml

```diff
@@ -80,7 +80,7 @@
         <native-maven-plugin.version>0.9.9</native-maven-plugin.version>
         <micronaut.version>3.3.0</micronaut.version>
         <micronaut.aot.version>1.0.0-M7</micronaut.aot.version>
-        <it.micronaut.version>3.2.8-SNAPSHOT</it.micronaut.version>
+        <it.micronaut.version>3.3.0-SNAPSHOT</it.micronaut.version>
```

Comment:
Could use `3.3.0` final I guess.

Reply:
Yeah, actually it should be 3.3.1-SNAPSHOT. The reason to use SNAPSHOT versions in IT is to either get upcoming parent/bom versions, and to ensure we test against the version that the plugin will debut with.